src/daemon/common.c

```diff
@@ -69,11 +65,15 @@ extern kstat_ctl_t *kc;
 #endif
 
 /* AIX doesn't have MSG_DONTWAIT */
-#ifndef MSG_DONTWAIT
+#if !defined(MSG_DONTWAIT) && defined(MSG_NONBLOCK)
 #define MSG_DONTWAIT MSG_NONBLOCK
 #endif
 
-#if !HAVE_GETPWNAM_R
+#ifdef WIN32
```

Comment:
Nit: Putting this into an else branch seems cleaner.

Reply:
Done.